bin/acala/service/src/chain_spec/mandala.rs

```diff
@@ -108,8 +108,8 @@ pub fn latest_mandala_testnet_config() -> Result<ChainSpec, String> {
 	let wasm_binary = mandala_runtime::WASM_BINARY.ok_or("Mandala runtime wasm binary not available")?;
 
 	Ok(ChainSpec::from_genesis(
-		"Acala Mandala TC5",
-		"mandala5",
+		"Acala Mandala PC1",
```

Comment:
we should have two of those, one for TC5, and one for PC2

Reply:
Could we use `acala-dev` for TC5?

Reply:
oh you are right.